handler.go

```diff
@@ -65,6 +72,11 @@ func WithOIDC(securitySchemeName, oidcURL string) Option {
 	}
 }
 
+// WithAddServerUrls adds a HandlerMiddleware which adds the requests host to the OpenAPI specs
+// server list, once with HTTP, once with HTTPS.
+//
+// This is meant to be used on APIs which could be served under any hostname and where
+// you don't want to hardcode the hostname in your spec.
 func WithAddServerUrls() Option {
```

Comment:
In retrospect,  shouldn't we make it singular (WithAddServerUrl()) + add a parameter to define the protocol? 🤔

Reply:
Hm, yeah. Strictly speaking that would already be a breaking change in this packages public API sooooooo :D 
Let's write an issue for that.

Reply:
I propose we discuss that further in #2 and keep that change outside of this PR.